Packages/KeyAppKit/Sources/BankTransfer/Striga/Repository/StrigaBankTransferUserDataRepository.swift

```diff
@@ -368,6 +368,10 @@ public final class StrigaBankTransferUserDataRepository: BankTransferUserDataRep
         try await remoteProvider.initiateSEPAPayment(userId: userId, accountId: accountId, amount: amount, iban: iban, bic: bic).challengeId
     }
 
+    public func availableRegions() -> [String] {
+        ["at", "be", "bg", "hr", "cy", "cz", "dk", "ee", "fi", "fr", "gr", "es", "nl", "is", "li", "lt", "lu", "lv", "mt", "de", "no", "pl", "pt", "ro", "sk", "si", "se", "hu", "it", "ch", "gb"]
```

Comment:
do you mind adding a comment that this is alpha2 ?

Reply:
Yeah, sure